lib/velocity-animate-shim.js

```diff
@@ -2,16 +2,17 @@
 // requires window. Note that this just no-ops the components so
 // that they'll render, rather than doing something clever like
 // statically rendering the end state of any provided animations.
-if (typeof window !== 'undefined') {
+if (process.env.NODE_ENV === 'test' ||  typeof window === 'undefined') {
```

Comment:
Yes,I agree. `process` is a Node.js global variable and may not available in browser env. I  fix this issue by judge whether `process` is exist